onnxruntime/core/graph/model.cc

```diff
@@ -91,6 +91,10 @@ Model::Model(std::unique_ptr<ModelProto> model_proto, const IOnnxRuntimeOpSchema
         " specifies which version of the ONNX OperatorSet is being imported.");
   }
 
+  if (!model_proto->has_ir_version() || model_proto->ir_version() > ONNX_NAMESPACE::Version::IR_VERSION) {
+      throw std::invalid_argument("Unknown model file format version.");
+  }
```

Comment:
I don't think this is inline with our requirements.

Reply:
this is from change ([https://github.com/microsoft/onnxruntime/pull/2413](https://github.com/microsoft/onnxruntime/pull/2413)) and we are just pulling in the latest version to our branch. The change is already in master. If it needs to be addressed, please open a bug to the person who changed this.

---
In reply to: [350342604](https://github.com/microsoft/onnxruntime/pull/2470#discussion_r350342604) [](ancestors = 350342604)

Reply:
@martinb35 : It's not in master yet. I'm happy to add const, but @yuslepukhin , I'm curious about what you mean regarding ORT requirements? Have you talked with Pranav and George Wu?